packages/npm/send/src/index.tsx

```diff
@@ -1,50 +1,22 @@
-import { NativeModules, Platform } from 'react-native'
+import { NitroModules } from 'react-native-nitro-modules'
+import type { Request, Send, SendResult } from './Send.nitro'
+export * from './Send.nitro'
```

Comment:
```suggestion
import type { Request as _Request, Send } from './Send.nitro'

export type Request = _Request & { method: Method }
export type SendResult = (Response & { result: 'success' }) | (SendError & { result: 'error' })
export { Parameters, Response, Code } from './Send.nitro'
```

We don't want to expose SendResult or Send as those are internal to the JSI interface.

Reply:
We also want to redefine the Request and SendResult types so we don't expose the internal/hacked versions